lib/kaur/result.ex

```diff
@@ -408,6 +450,42 @@ defmodule Kaur.Result do
     keep_if(result, &(not predicate.(&1)), error_message)
   end
 
+  @doc ~S"""
+  Replace the error of a result tuple.
```

Comment:
Shouldn't we use `Replaces` here to stay consistant with the rest of the doc? (3rd person everywhere else).